src/config.rs

```diff
@@ -7,6 +14,10 @@ use rustls::{ClientConfig, ConfigBuilder, WantsVerifier};
 /// This adds methods (gated by crate features) for easily configuring
 /// TLS server roots a rustls ClientConfig will trust.
 pub trait ConfigBuilderExt {
+    /// Use the platform's native verifier to verify server certificates.
```

Comment:
WDYT about putting a link to the platform verifier crate here? I think it might be non-obvious what platforms are supported, or what the various per-platform capabilities are. Duplicating that info here will be problematic, but we could at least give a pointer to the platform verifier repo/docs.

Reply:
Done.